plugins/woocommerce/includes/class-wc-tracker.php

```diff
@@ -568,7 +584,21 @@ private static function get_orders_origins() {
 
 		$orders_by_origin = array();
 		foreach ( $orders_origin as $origin ) {
-			$orders_by_origin[ $origin->origin ] = (int) $origin->count;
+			$key = strtolower( $origin->origin );
+
+			// If the origin name has a unique id, discard it, especially if the count is 1.
+			if ( 1 === $origin->count ) {
```

Comment:
```suggestion
			if ( 1 == $origin->count ) {
```
In my test environment, `$origin->count` is a string, so it never `===` 1.

Reply:
Thanks Dan. I've added a commit to use loose comparison.